api/src/test/java/com/epam/pipeline/test/creator/docker/DockerCreatorUtils.java

```diff
@@ -199,6 +212,17 @@ public static Tool getTool(final String owner) {
         return getTool(ID, owner);
     }
 
+    public static ToolGroup getToolGroup(final Long id, final String name, final Long registryId, final String owner) {
+        final ToolGroup toolGroup = new ToolGroup();
+        if (id != null) {
```

Comment:
It seems that this line is redundant: if `id` is `null` `toolGroup` will set `null`

Reply:
Done!